pkgs/development/tools/build-managers/waf/setup-hook.sh

```diff
@@ -38,15 +42,11 @@ wafConfigurePhase() {
     runHook postConfigure
 }
 
-if [ -z "${dontUseWafConfigure-}" -a -z "${configurePhase-}" ]; then
-    configurePhase=wafConfigurePhase
-fi
-
 wafBuildPhase () {
     runHook preBuild
 
     # set to empty if unset
-    : ${wafFlags=}
+    : "${wafFlags=}"
```

Comment:
Are you sure about this?

Reply:
Shellcheck complained about the previous form.

Reply:
> Shellcheck complained about the previous form.

In general, quoting or not quoting variables makes a difference when these are command arguments. But in this case (just read some Bash docs), it doesn't matter.